Server/Plugins/APIDump/APIDesc.lua

```diff
@@ -437,6 +437,22 @@ return
 			},
 		}, -- cBlockInfo
 
+		cChannelManager =
+		{
+			Desc = [[
+				This class allows plugins to register handlers for specific plugin channels.
+			]],
+
+			Functions =
+			{
+				AddClientToChannel = { Params = "{{cClientHandle}}, Channel Name", Notes = "Adds and registers a client on the specified channel." },
+				RemoveClientFromChannel = { Params = "{{cClientHandle}}, Channel Name", Notes = "Removes and unregisters a client on the specified channel." },
+				RegisterChannel = { Params = "Channel Name, Callback", Return = "bool", Notes = "Registers a handler for a specific channel. Returns true if it was able to be registered." },
+				RemoveChannel = { Params = "Channel Name", Return = "bool", Notes = "Removes the handler for the specified channel. Returns true if the handler was removed." },
+				BroadcastChannelMessage = { Params = "Channel Name, Message, {{cWorld|World}}", Notes = "Sends the message on the specified channel. Limits the message to the specified world, if provided." }
+			}
+		}, -- cChannelManager
```

Comment:
I find the API documentation rather lacking. The function descriptions don't tell much about what they are used for. I'd appreciate an explanation (why do I need to add a client to a channel?) and, best of all, a piece of an example code.